packages/web-react/src/components/ScrollView/ScrollView.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useRef } from 'react';
+import classNames from 'classnames';
+import { SpiritScrollViewProps } from '../../types';
+import { useStyleProps } from '../../hooks';
+import { useScrollViewStyleProps } from './useScrollViewStyleProps';
+import { useScrollPosition } from './useScrollPosition';
+import { SCROLL_VIEW_DEFAULT_EDGE_INDICATOR, SCROLL_VIEW_DEFAULT_DIRECTION } from './constants';
+
+const ScrollView = (props: SpiritScrollViewProps) => {
+  const {
+    children,
+    direction = SCROLL_VIEW_DEFAULT_DIRECTION,
+    indicators = SCROLL_VIEW_DEFAULT_EDGE_INDICATOR,
```

Comment:
I prefer using the `edge` prefix. I was asking by myself, what kind of indicators are they or what they indicate.

```suggestion
    edgeIndicators = SCROLL_VIEW_DEFAULT_EDGE_INDICATOR,
```

Reply:
But this is maybe also for @adamkudrna and @crishpeen because I did not notice it before that this is based on class names in the ScrollView component. I have noticed that in many places where there is a usage of `indicators` vs `EDGE_INDICATORS`. And it makes me no sense when we assign edge indicator to only indicator. Because there is an information loss. What do you think?

Reply:
I had this exactly set there, but when I looked at twig, I changed it to indicators

Reply:
done

Reply:
I do not blame you for anything. The problem I see is that in Twig there is no usage of `edge`. I like the name "edge indicator" and I would like to prefer it over just "indicator". The problem I see here is that assign one to another. E.g. I would rather use `indicators = SCROLL_VIEW_DEFAULT_INDICATOR` or `edgeIndicators = SCROLL_VIEW_DEFAULT_EDGE_INDICATOR`. But not both. But I repeat, this is only nitpicking and thinking about naming consistency.